vision/common/constants.py

```diff
@@ -9,6 +9,9 @@
 # single channel image of booleans
 Mask: TypeAlias = NDArray[Shape["*, *"], Bool]
 
+# A kernel for image processing steps
+Kernel: TypeAlias = NDArray[Shape["*, *"], UInt8]
```

Comment:
Does this need to be a type alias? By abstracting like this, you are making the shape arbitrary, which reduces specificity. A 3x3 kernel will be of shape "3, 3", but having a generic `Kernel` type removes that detail.